cdap-docs/admin-manual/source/operations/operations-dashboard.rst

```diff
@@ -0,0 +1,60 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2014-2017 Cask Data, Inc.
+
+.. _operations-dashboard:
+
+======================
+Operations Dashboard
+======================
+
+Operations dashboard provides users two views: Dashboard and Reporting.
+
+The Dashboard view is a real-time interactive interface that allows users to take a sneak peek
+of the program runs statistics, resource usage within any 24 hours in the last week.
+A user can also select any hour within the 24 hours view to get program run details within that hour.
+
+The Reporting view provides a comprehensive report of program runs, users can generate a report to view resource usage
+and other statistics within a given time range such as 24 hours or one week. A Spark batch
+job will process the program runs and generate the report for the user.
+
+
+Operations Dashboard Usage
+==========================
+
+Operations dashboard provides operational view for pipeline and CDAP application programs.
+CDAP does not offer a lot today in terms of a holistic view of all pipelines on a CDAP installation.
+For monitoring and debugging purposes,
+CDAP has logs and metrics aggregation and service capabilities as a platform.
+However, they are not presented in the most consumable manner for administrators and operators on the CDAP UI.
+CDAP Operations dashboard would be a useful tool to answer the following use cases and questions by administrators.
+
+  - CDAP System status
+      - Filtering and viewing pipelines that are currently running
+      - Filtering and viewing pipelines that have been submitted but are not yet running in Cluster
+      - Slicing and dicing by time, program-status, namespace, user, start-method etc.
+      - Easily monitor which pipelines have failed in the last 24 hours and at which time did the failures happen.
+
+  - Looking into the future pipeline resource usage
+      - Operations dashboard also provides a view of the programs that are scheduled to be run in the future,
+      - By peeking into future time range in dashboard, users can look into how many pipelines/programs will be run each hour, enabling them to predict and manage cluster usage efficiently.
+
+
+Reporting Application Usage
+===========================
+
+Reporting offers tool for administrators to take a historical look at their cluster's resource usage, program runs and provides various
+filters that they could apply to gain insight into the pipelines and CDAP applications performance
+and resource utilization.
+
+Example usage
+  - Look into pipelines that failed in the last week as a table
+  - Reports result contain two components, summary and program runs table.
+  - In authorization enabled clusters, administrators can generate report to summarize pipeline runs by users and
```

Comment:
you mean authentication?
Only authentication is the requirement, and not authorization, right?

Reply:
its authorization enabled cluster as its principal name we group by.

Reply:
so, it's impersonation, which requires authentication, but not authorization.